api/ws/rpc/connect.js

```diff
@@ -121,14 +121,19 @@ const connectSteps = {
 		// Register RPC methods on peer
 		peer = _.reduce(
 			wsServer.endpoints.rpc,
-			(peerExtendedWithRPC, localHandler, rpcProcedureName) => {
+			(peerExtendedWithRPC, _localHandler, rpcProcedureName) => {
```

Comment:
Why is `localHandler` changed to `_ localHandler`? I don't see any changes that would re-use that variable here. Please don't use underscores.

Reply:
when a variable is not used or if its an optional parameter, in most programming(that I know of) use `_` as convention and even `eslint` says the same https://eslint.org/docs/rules/no-unused-vars#argsignorepattern
so what @diego-G has done is correct.

Reply:
https://eslint.org/docs/rules/no-unused-vars#argsignorepattern

Reply:
I think when the argument is not used, `_` is often use but putting the underscore in front is different thing. In this case, it should've been `(peerExtendedWithRPC, _, rpcProcedureName) => {`
However, if you use lodash it conflicts.

FYI, in elements, we decided to use `_` prefix for private member in class, but in othercases we never use `_` prefix.
If it's non-used argument, we use (input1, _, input3)

Reply:
@shuse2 if we are following `eslint` conventions, shouldn't we stick to the conventions? here https://eslint.org/docs/rules/no-unused-vars#argsignorepattern they clearly say use `_` in front of the variable, however, I agree in some other languages only `_` is used when the variable is not used, but as `eslint` suggest we can stick to `_` with variable name convention rather than our own.

Reply:
I think this rule is only suggesting you can setup the pattern to ignore, and this is usable if you setup like `/*eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]*/`.
Also, I think by default, if second parameter is not used but third parameter is used, then it shouldn't complain

Reply:
What it is clear is that having a tag to know if a var is used or not is something suggested by eslint itself. IMHO having just an underscore is meaningless. Eslint suggests by default just adding an underscore at the beginning. I don't see the big deal. It is just a convention we can adopt or not. I've done it because the IDE suggests me to do it and I've found it useful.

Reply:
either only `_` or `_` prefixed with name, both patterns are fine with me, it's only a choice now to pick which one.

Reply:
@MaciejBaj in any case, this is not related to this PR. If it really bothers you I can remove it from here and from any other PR, and I will open another proper PR just for the case.

Reply:
@diego-G @ManuGowda my question https://github.com/LiskHQ/lisk/pull/2589/files#r237434813 still remains not answered. I still don't know why you decided to change this very one `localHandle` variable name in this PR.

> when a variable is not used or if its an optional parameter, in most programming(that I know of) use `_` as convention and even `eslint` says the same https://eslint.org/docs/rules/no-unused-vars#argsignorepattern
> so what @diego-G has done is correct.

@ManuGowda you are referring to Eslint rule which wasn't accepted to implement as part of  https://github.com/LiskHQ/lisk/issues/1597, following the airbnb standards.





> What it is clear is that having a tag to know if a var is used or not is something suggested by eslint itself. IMHO having just an underscore is meaningless. Eslint suggests by default just adding an underscore at the beginning. I don't see the big deal. It is just a convention we can adopt or not. I've done it because the IDE suggests me to do it and I've found it useful.

Part of the issue #1597 you're solving now is applying the rule: ` "no-underscore-dangle": 161`, which makes the declaration `let _localHandler = "LH"` invalid, and `let _ = "LH"` valid. The reference - https://eslint.org/docs/rules/no-underscore-dangle.

To reiterate:
- Don't implement the rule `/*eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]*/`, which conflicts with rule "no-underscore-dangle", which agreed as part of #1597.
- Don't include random changes of other eslint rules in this (`Apply eslint-rule no-nested-ternary`) PR.